docker/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Artemis-Prod-MySQL Setup
+# ----------------------------------------------------------------------------------------------------------------------
+
+# this links to /docker/artemis-prod-mysql.yml which is the default artemis production docker compose setup
+# just using a symlink doesn't work because of the relative paths ;) (TODO: recheck this after the refactoring to .zs/docker)
```

Comment:
Not sure if we should use symlinks in the project or not. Don't know how much of a problem this is for windows users?!

Reply:
We don't use symlinks in the project, as Windows users have to configure git to know about symlinks.